source/intro_release_notes/release_notes/whats_new.rst

```diff
@@ -4,12 +4,26 @@
 What's New in 6.6
 ================================================================================
 
-**OpenNebula 6.6 ‘Electra’** is the fourth stable release of the OpenNebula 6 series. This new release comes packed with new functionality, mostly oriented to aid day-2 operations on production deployments of OpenNebula. There are two significant campaigns we would like to highlight in this regard. 'Electra' comes with an exciting integration with `Prometheus <https://prometheus.io/>`__, which includes packaging of a pre-configured Prometheus instance with metrics tailored for the optimal observability of an OpenNebula cloud. This integration also includes Prometheus Alert Manager with predefined alarms that can be enabled to react to issues with OpenNebula operations. And last but not least, a set of 3 (lush!) dashboards for `Grafana <https://grafana.com>`__, the open observability platform.
```

Comment:
We think it is better to keep the first part in a more descriptive way rather than adding a subsection for each feature

Reply:
So it is better to revert changes on this file